packages/sanity/src/core/form/store/formState.ts

```diff
@@ -492,7 +492,7 @@ function prepareObjectInputState<T>(
     return null
   }
 
-  const readOnly =
+  const objectIsReadOnly =
```

Comment:
Renamed this variable to make it clear that it's the parent object's readOnly state